tests/test-data/projects/docker-compose/basic/service2/Dockerfile-alt

```diff
@@ -0,0 +1 @@
+alternative Dockerfile (basic/service2)
```

Comment:
Does it matter that there's no EOL here?

Reply:
I will add a line break, 👍  but this Dockerfile is invalid, of course.